tests/e2e/stepDefinitions/presentationViewerContext.js

```diff
@@ -25,7 +29,12 @@ Given('user {string} has logged in', async function (user) {
 When(
   'user {string} previews a markdown file {string} in presentation viewer',
   async function (user, fileName) {
-    await ocis.openMDFileInPresentationViewer()
+    if (config.targetApp === 'opencloud') {
+      const opencloud = new Opencloud()
+      await opencloud.openMDFileInPresentationViewer()
+    } else {
+      await ocis.openMDFileInPresentationViewer()
+    }
```

Comment:
we can have new PO called `FilesPage` and all the conditional logics can go there. Then in the step def we should only import FilesPage
```suggestion
      await filesPage.openMDFileInPresentationViewer()
```